### ISSUE TITLE
Mining 2 coins but panel only shows one pool/coin being mined.

### ISSUE THREAD

**Issue by ayswift** [UnamSanctam/UnamWebPanel#73]
Hello! I've installed your panel and it's quite amazing I must say but I want to report one bug. I can see from the checker and from nanopool that i am successfully mining 2 coins (XMR + ETC) but on the panel its only showing XMR. Maybe you just didn't implement it to show 2 coins or something but wanted to point it out .

**Comment by UnamSanctam** [comment]
No it will show both, run the checker and make sure that the endpoint URL is correct (the same) in both of them. You can post what the checker returns here as well if you want (you can censor parts of it).

**Comment by ayswift** [comment]
![image](https://user-images.githubusercontent.com/98239336/151250690-cfdef0ea-cab6-49a0-b0da-4ce24a2a2094.png)

here, im 100% sure both endpoint urls are the same

**Comment by UnamSanctam** [comment]
Hmm yes then both should show up, they are added according to their type (xmrig or ethminer). Does your host have some sort of anti-DDoS filter that could be blocking one of the connections? Since if you run a local webserver with the web panel and connect to that you'll see both of them appear.

**Comment by ayswift** [comment]
Nah, Im using Dreamhost's cheapest plan and they have no ddos prot i believe

**Comment by UnamSanctam** [comment]
Can you try running it without the XMR miner to see if the ETH miner appears? Since in that case the webhost is blocking the multiple connections.

**Comment by ayswift** [comment]
course, ill do that now

**Comment by ayswift** [comment]
Okay, ive been waiting for 3-4 min now and nothing is showing up in the panel when i only use ETC... I can see its mining thru Nanopool + checker but nothing in ur panel

**Comment by ayswift** [comment]
seems like a bug right? i tried eth/etc etc and only xmr shows even when theres 1 running

**Comment by UnamSanctam** [comment]
No it's most likely your host doing it but I'm not sure why it's discriminating between the two calls, since they are nearly identical in structure.

**Comment by ayswift** [comment]
but how would it be my host lol. any way i can disable whatever they are doing?

**Comment by UnamSanctam** [comment]
Not sure since I've never tried Dreamhost but you can try any free webhost (or host it yourself) temporarily to see if both appear.

**Comment by ayswift** [comment]
ye okay

**Comment by ayswift** [comment]
yo, i uploaded it to 000hosting about 3 times now and when i try to login it just says "error, something went wrong"
any ideas?

**Comment by UnamSanctam** [comment]
Upgrade the PHP to 7.4 or above: https://www.000webhost.com/forum/t/how-to-change-upgrade-or-downgrade-php-version-of-my-website/39336.

**Comment by ayswift** [comment]
okay yeah ur right lol, the host was the issue.
